### PR TITLE
Add multiple token into gallery editor

### PR DESCRIPTION
**Pull request by jakzaizzat**
### Summary of Changes

- Add an ability to add multiple token into gallery editor

### Demo or Before/After Pics

https://github.com/gallery-so/gallery/assets/4480258/ce7ff2ed-5808-423c-9a60-fe89dc3b39b4

### Edge Cases

1. Add single token and group token at the same times

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [x] I've tested the changes and all tests pass.
- [x] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/NftSelector/NftSelectorContract/NftSelectorContractHeader.tsx [hunk 1]
```diff
@@ -47,7 +54,7 @@ export function NftSelectorContractHeader({
           {title}
         </Typography>
       </View>
-      {rightButton ? <View>{rightButton}</View> : null}
+      <Animated.View style={animateStyle}>{rightButton && rightButton}</Animated.View>
```
Comment (Robinnnnn) [inline apps/mobile/src/components/NftSelector/NftSelectorContract/NftSelectorContractHeader.tsx:57]: 🤔 `rightButton && rightButton`?

File: apps/mobile/src/screens/GalleryScreen/GalleryEditorNftSelector.tsx [hunk 2]
```diff
@@ -29,20 +35,98 @@ export function GalleryEditorNftSelector() {
...
+            ];
+          } else {
+            return [
+              ...prevTokens,
+              {
+                id: tokenId,
+              },
+            ];
+          }
+        });
```
Comment (Robinnnnn) [inline apps/mobile/src/screens/GalleryScreen/GalleryEditorNftSelector.tsx:64]: depending on the number of tokens, might be faster to store `selectedTokens` as a `Set` instead of an array. will need to convert both ways though

File: apps/mobile/src/components/NftSelector/NftSelectorContract/NftSelectorContractHeader.tsx (hunk 1 above)
Comment (jakzaizzat) [reply apps/mobile/src/components/NftSelector/NftSelectorContract/NftSelectorContractHeader.tsx]: Thats weird. Removed it!

File: apps/mobile/src/screens/GalleryScreen/GalleryEditorNftSelector.tsx (hunk 2 above)
Comment (jakzaizzat) [reply apps/mobile/src/screens/GalleryScreen/GalleryEditorNftSelector.tsx]: That smart! Its getting messy with the array. I've replaced it to used `Set`

**Review by Robinnnnn — APPROVED** [review]
